integration/search-page-uat.spec.js

```diff
@@ -21,7 +21,7 @@ context('Search page', () => {
 
   // scope is present with no profile
   it("Filter should appear if scope is present, but if its value is not 'qld.gov.au'", () => {
-    cy.visit(`${ROOT_URL}/?query=rego&collection=${search_collection}&profile=disaster&scope=tmr.qld.gov.au&collection=${search_collection}`)
```

Comment:
Does the same need to be done on qg-web-template also?

Reply:
Yes. We need to remove num_ranks, tiers and scope from SWE as well. But it won't be breaking if they exist. Manjeet advised it can be done later.